HTTP_API.md

```diff
@@ -1852,6 +1857,6 @@ The handler-specific descriptions below describe how each handler populates the
 
     ##### example
     ```
-    $ curl -v --data "{\"myKey\":\"updatedValue\",\"another-key\":\"another-updated-value\",\"new-key\":\"new-value\"}" http://localhost:8181/api/beta/recordings/localhost_myRecording_20220309T203725Z.jfr/metadata/labels
+    $ curl -v --data "{\"myKey\":\"updatedValue\",\"another-key\":\"another-updated-value\",\"new-key\":\"new-value\"}" http://localhost:8181/api/beta/recordings/localhost%3A0/localhost_myRecording_20220309T203725Z.jfr/metadata/labels
```

Comment:
Looks like this example got overlooked. It still says `/api/beta`, but it's `/api/v2` in the outline above.